api/src/main/resources/metadata/Custom_Concepts.xml

```diff
@@ -4321,6 +4402,35 @@
+    <concept_answer  concept_answer_id="7237"  concept_id="165244"  answer_concept="165238"   creator="1"  date_created="2019-09-05 11:48:00"  sort_weight="7"  uuid="0a3622b1-b2bd-4a7c-9866-ef7209b6c63f"/>
+    <concept_answer  concept_answer_id="7238"  concept_id="165244"  answer_concept="165237"   creator="1"  date_created="2019-09-05 11:48:00"  sort_weight="6"  uuid="5b94fc24-bf40-479f-8e90-2d03a991822a"/>
+    <concept_answer  concept_answer_id="7239"  concept_id="165244"  answer_concept="90150"   creator="1"  date_created="2019-09-05 11:48:00"  sort_weight="5"  uuid="b7c97190-a534-4c61-ac20-75af7d3c7692"/>
+    <concept_answer  concept_answer_id="7240"  concept_id="165244"  answer_concept="165243"   creator="1"  date_created="2019-09-05 11:48:00"  sort_weight="12"  uuid="d6a30ff6-119d-4819-af54-6104f02dd095"/>
+    <concept_answer  concept_answer_id="7241"  concept_id="165244"  answer_concept="165242"   creator="1"  date_created="2019-09-05 11:48:00"  sort_weight="11"  uuid="7d3b93e7-0d5a-4d75-9b82-b00e5edb28da"/>
+    <concept_answer  concept_answer_id="7242"  concept_id="165244"  answer_concept="90118"   creator="1"  date_created="2019-09-05 11:48:00"  sort_weight="2"  uuid="02debf49-5c20-4692-98bb-3e901fedb5b3"/>
+    <concept_answer  concept_answer_id="7243"  concept_id="165244"  answer_concept="165236"   creator="1"  date_created="2019-09-05 11:48:00"  sort_weight="4"  uuid="ab13f044-73fa-422d-95b5-0fbf63536b66"/>
+    <concept_answer  concept_answer_id="7244"  concept_id="165244"  answer_concept="165241"   creator="1"  date_created="2019-09-05 11:48:00"  sort_weight="10"  uuid="c4ee2e5f-40df-4de0-96ef-bb7c4a0e0596"/>
+    <concept_answer  concept_answer_id="7245"  concept_id="165244"  answer_concept="165240"   creator="1"  date_created="2019-09-05 11:48:00"  sort_weight="9"  uuid="9f9d43f6-ddca-472f-8f5e-eeb04d577715"/>
+    <concept_answer  concept_answer_id="7246"  concept_id="165244"  answer_concept="165239"   creator="1"  date_created="2019-09-05 11:48:00"  sort_weight="8"  uuid="f5458595-fcd4-46d1-9872-01256d0d0c09"/>
+    <concept_answer  concept_answer_id="7247"  concept_id="165244"  answer_concept="165234"   creator="1"  date_created="2019-09-05 11:48:00"  sort_weight="1"  uuid="bd454f0d-b504-4241-a7dd-bf7b6f6ff1e3"/>
+    <concept_answer  concept_answer_id="7248"  concept_id="165244"  answer_concept="165219"   creator="1"  date_created="2019-09-05 11:53:35"  sort_weight="13"  uuid="0ebd42d3-019d-428e-8870-1d1792b07296"/>
+    <concept_answer  concept_answer_id="7249"  concept_id="165244"  answer_concept="165223"   creator="1"  date_created="2019-09-05 11:53:35"  sort_weight="14"  uuid="eff0f696-8316-44a7-bfdc-38468c24a33f"/>
+
+    <!--    ART SUMMARY PAGE CONCEPT ANSWERS 2019-->
+    <concept_answer  concept_answer_id="7229"  concept_id="165169"  answer_concept="165224"    creator="1"  date_created="2019-09-03 16:58:33"  sort_weight="13"  uuid="04a4b004-5a8f-4764-979c-6bddc5c8d781"/>
```

Comment:
@solemabrothers  please change the concept_id field from 165169 to 90200 as this answer is for care entry point and not for special category